at_client/src/main/java/org/atsign/client/util/RegisterUtil.java

```diff
@@ -108,34 +109,53 @@ public Boolean registerAtsign(String email, AtSign atsign, String registrarUrl,
      * @throws IOException
      * @throws AtException
      */
-    public String validateOtp(String email, AtSign atsign, String otp, String registrarUrl, String apiKey)
+    public String validateOtp(String email, AtSign atsign, String otp, String registrarUrl, String apiKey,
+            boolean confirmation)
```

Comment:
This is a breaking change. Please make `confirmation` an optional parameter. Customer is directly using this method.